colrev/packages/genai/src/genai_prescreen.py

```diff
@@ -154,15 +114,20 @@ def run_prescreen(
                 {
                     "Record": record.get_data()["ID"],
                     "Inclusion/Exclusion Decision": (
-                        "Included" if response.included else "Excluded"
+                        "Included" if prescreen_decision.included else "Excluded"
                     ),
-                    "Explanation": response.explanation,
+                    "Explanation": prescreen_decision.explanation,
                 }
             )
 
-        print(f"\nGenAI (model: {self.settings.model}) screening decisions:")
-        self._print_table(screening_decisions)
-        print("\n")
+        self.review_manager.paths.prescreen.mkdir(parents=True, exist_ok=True)
```

Comment:
@geritwagner can you check this? I moved the screening decision table from a CLI print into a csv output instead. Is this the right way to create the file here?